lib/node_modules/@stdlib/stats/base/dists/geometric/variance/benchmark/benchmark.native.js

```diff
@@ -24,7 +24,7 @@ var resolve = require( 'path' ).resolve;
 var bench = require( '@stdlib/bench' );
 var Float64Array = require( '@stdlib/array/float64' );
 var tryRequire = require( '@stdlib/utils/try-require' );
-var randu = require( '@stdlib/random/base/randu' );
+var uniform	= require( '@stdlib/random/base/uniform' );
```

Comment:
```suggestion
var uniform = require( '@stdlib/random/base/uniform' );
```
Hey there should have been space instead of tab here.